include/z64actor.h

```diff
@@ -164,8 +164,10 @@ typedef struct ActorShape {
 //
 #define ACTOR_FLAG_16 (1 << 16)
 
-//
-#define ACTOR_FLAG_17 (1 << 17)
+// Actor will be influenced by the pitch (x rot) of Player's left hand when being carried.
+// This is different from the deafult actor carry behavior, which influences the actor's yaw.
```

Comment:
```suggestion
// This is different from the default actor carry behavior, which influences the actor's yaw.
```

does this mean the yaw is unaffected with this flag?

Reply:
Correct. Its assumed that the actor is so heavy that link wont be able to turn, I guess